h/static/scripts/group-forms/components/CreateEditGroupForm.tsx

```diff
@@ -136,7 +136,18 @@ export default function CreateEditGroupForm() {
     'unmodified' | 'unsaved' | 'saving' | 'saved'
   >('unmodified');
 
+  // Location is set after creating a group, but not after editing it.
+  // We can't redirect as part of `onSubmit`, because other pieces of state have
+  // not been applied yet, causing useWarnOnPageUnload to display its warning
+  const [windowLocation, setWindowLocation] = useState<string>();
+  useEffect(() => {
+    if (windowLocation) {
+      setLocation(windowLocation);
```

Comment:
`setLocation` is a helper function which looks like this:

```
export function setLocation(newLocation: string): void {
  window.location.href = newLocation;
}
```

However, it sounds like a state updater, so maybe we should rename it to `updateLocation` or something like that.